.github/workflows/main.yml

```diff
@@ -2,28 +2,28 @@ name: CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
   test:
     name: Test on node ${{ matrix.node_version }} and ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node_version: ['10.x', '12.x', '14.x', '15.x', '16.x']
+        node_version: ["10.x", "12.x", "14.x", "15.x", "16.x"]
```

Comment:
Is there any intentional change in this file? If not, best to leave the file unmodified, to avoid adding noise to the diff.

Same with `tsconfig.json`.

Reply:
Oh, I didn't notice changed in `.github/workflows/main.yml` and in `tsconfig.json`. They must've been caused by running `npm run format` before final testing and committing.

I'm reverting changes in the two files.